packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -384,6 +401,26 @@ class _CodeViewState extends State<CodeView>
                       },
                     ),
                     const SizedBox(width: denseSpacing),
+                    ValueListenableBuilder<bool>(
+                      valueListenable:
+                          widget.codeViewController.showProfileInformation,
+                      builder: (context, showProfileInformation, _) {
+                        if (!showProfileInformation) {
+                          return const SizedBox();
+                        }
+                        return Row(
+                          children: [
+                            ProfileInformationGutter(
+                              scrollController: profileController!,
+                              lineCount: lineCount,
+                              lineOffset: lineOffset,
+                              sourceReport: sourceReport,
+                            ),
+                            const SizedBox(width: denseSpacing),
```

Comment:
use a Padding widget instead of a Row if we are just using this SizedBox to pad on the right